mithril-common/src/entities/epoch_settings.rs

```diff
@@ -19,4 +19,7 @@ pub struct EpochSettings {
 
     /// Signers that will be able to sign on the next epoch
     pub next_signers: Vec<Signer>,
+
+    /// Cardano transactions signing configuration for the current epoch
+    pub current_cardano_transactions_signing_config: Option<CardanoTransactionsSigningConfig>,
```

Comment:
Maybe we should name that field `next_cardano_transactions_signing_config` :thinking: ?

Reply:
Indeed, in this case, should I also prepare the EpochSettings structure by creating the field `current_cardano_transactions_signing_config`, which will contain the configuration of the previous epoch in the next PR?

Reply:
@Alenar @sfauvel WDYT?

Reply:
Need to update to `cardano_transactions_signing_config` and `next_cardano_transactions_signing_config`.

Reply:
Done ✅